README.md

```diff
@@ -1 +1,60 @@
-# my-portfolio
\ No newline at end of file
+# my-portfolio
+![](https://img.shields.io/badge/myapp-blueviolet)
+
+
+> Personal portfolio project.
+
+<!-- ![screenshot](./app_screenshot.png) -->
```

Comment:
- [ ] **OPTIONAL** Please keep in mind that it is always better if the README file contains a screenshot of the project. It keeps it way more professional and good-looking for future viewers.
You can always refer to this [template](https://github.com/microverseinc/readme-template)